lib/content-services/src/lib/search/components/search-filter-chips/search-widget-chip/search-widget-chip.component.html

```diff
@@ -20,31 +20,33 @@
 </mat-chip>
 
 <mat-menu #menu="matMenu" backdropClass="adf-search-filter-chip-menu" (closed)="onClosed()">
-    <div #menuContainer [attr.data-automation-id]="'search-field-' + category.name">
-        <adf-search-filter-menu-card (click)="$event.stopPropagation()"
-                                     (keydown.tab)="$event.stopPropagation();"
-                                     (close)="menuTrigger.closeMenu()">
+    <div role="menuitem">
```

Comment:
How is adding `div` with `menuitem` role related to 'Group of form controls not associated with group label'?

Reply:
Mat-menu-panel having role menu which is parent div and aria-roles must contain particular children so we add role menuitem as children div in mat-menu-content to create sort of relation between them and it convey group label to form control as menuitem to user.

Reply:
Which label it conveys? It doesn't have any label applied, also just taking all menu content and wrapping it inside one `menuitem` div sounds werid to me, did you check how it works with screen reader?